File: backend/src/main/java/com/demo/student_management/dto/hocsinh/HocSinhSummaryResponse.java
package com.demo.student_management.dto.hocsinh;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class HocSinhSummaryResponse {

    private String idHocSinh;

    private String ten;

    private String idLop;

    private Double tbHocKy1;

    private Double tbHocKy2;
}

File: frontend/StudentManagement.Shared/Dtos/HocSinh/HocSinhSummaryResponse.cs
namespace StudentManagement.Shared.Dtos.HocSinh;

public class HocSinhSummaryResponse
{
    public string IdHocSinh { get; set; } = string.Empty;
    public string Ten { get; set; } = string.Empty;
    public string IdLop { get; set; } = string.Empty;
    public double? TbHocKy1 { get; set; }
    public double? TbHocKy2 { get; set; }
}
